homeassistant/components/camera/synology.py

```diff
@@ -203,48 +92,48 @@ def camera_image(self):
     @asyncio.coroutine
     def async_camera_image(self):
         """Return a still image response from the camera."""
-        image_url = SYNO_API_URL.format(
-            self._synology_url, WEBAPI_PATH, self._camera_path)
-
-        image_payload = {
-            'api': CAMERA_API,
-            'method': 'GetSnapshot',
-            'version': '1',
-            'cameraId': self._camera_id
-        }
-        try:
-            with async_timeout.timeout(self._timeout, loop=self.hass.loop):
-                response = yield from self._websession.get(
-                    image_url,
-                    params=image_payload
-                )
-        except (asyncio.TimeoutError, aiohttp.ClientError):
-            _LOGGER.error("Error fetching %s", image_url)
-            return None
-
-        image = yield from response.read()
-
-        return image
+        return self._surveillance.get_camera_image(self._camera_id)
```

Comment:
is that async?

Reply:
No this is synchronous call using requests.get()

Reply:
Please call this in sync context. Remove all async stuff from this function decleration

Reply:
Updated